.circleci/config.yml

```diff
@@ -31,8 +31,6 @@ workflows:
     jobs:
       - build:
           filters:
-            branches:
-              only: master
```

Comment:
So this was the reason for builds not getting triggered at all :man_facepalming: 
Thanks!